blueprints/duck/index.js

```diff
@@ -0,0 +1,19 @@
+const entityNameLocals = require('../util').entityNameLocals;
```

Comment:
Either you place semicolons everywhere or omit them entirely.
You can define eslint rule about semicolons and use `eslint ---fix` to make it consistent everywhere